connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -464,8 +464,11 @@ private void convertMessages(ConsumerRecords<byte[], byte[]> msgs) {
         for (ConsumerRecord<byte[], byte[]> msg : msgs) {
             log.trace("{} Consuming and converting message in topic '{}' partition {} at offset {} and timestamp {}",
                     this, msg.topic(), msg.partition(), msg.offset(), msg.timestamp());
-            SchemaAndValue keyAndSchema = keyConverter.toConnectData(msg.topic(), msg.key());
-            SchemaAndValue valueAndSchema = valueConverter.toConnectData(msg.topic(), msg.value());
+            final SchemaAndValue keyAndSchema;
+            final SchemaAndValue valueAndSchema;
+
+            keyAndSchema = toConnectData(keyConverter, "key", msg, msg.key());
+            valueAndSchema = toConnectData(valueConverter, "value", msg, msg.value());
```

Comment:
How about just declaring and initializing each variable in one statement?

    SchemaAndValue keyAndSchema = toConnectData(keyConverter, "key", msg, msg.key());
    SchemaAndValue valueAndSchema = toConnectData(valueConverter, "value", msg, msg.value());

Also, in this case I'm not sure `final` adds much if any value.

Reply:
@rhauch, Fixed review comments. It's true `final` doesn't add much value in this case..Removed it. Thank you for the review comments..